spec/jobs/scheduled/create_holiday_events_spec.rb

```diff
@@ -31,13 +31,13 @@
     freeze_time Time.zone.local(2019, 8, 1)
     DiscourseCalendar::CreateHolidayEvents.new.execute(nil)
 
-    expect(CalendarEvent.pluck(:region, :description, :start_date, :username)).to match_array(
+    expect(CalendarEvent.pluck(:region, :description, :start_date, :user_id)).to match_array(
```

Comment:
Lots of changes here, but it' because I switched to relying on `username` and instead used `user_id`. Feels safer.